omnigan/trainer.py

```diff
@@ -641,13 +652,9 @@ def get_translation_loss(self, multi_domain_batch):
             fake_s = self.G.decoders["s"](fake_z).detach()
             real_s_labels = torch.argmax(self.G.decoders["s"](real_z).detach(), 1)
             mask = (
-                torch.randint(0, 2, real_s_labels.shape)
-                .to(torch.float32)
-                .to(self.device)
+                torch.randint(0, 2, real_s_labels.shape).to(torch.float32).to(self.device)
             )  # TODO : load mask
-            update_loss = (
-                self.losses["G"]["t"]["sm"](fake_s, real_s_labels) * mask
-            ).mean()
+            update_loss = (self.losses["G"]["t"]["sm"](fake_s, real_s_labels) * mask).mean()
```

Comment:
what does sm stand for?

Reply:
let me rephrase: what does `sm` mean *in this context*?

Reply:
No idea what it means. I've basically ignored everything to do with translation here so I missed this

Reply:
ouuuuh semantic matching loss